lib/model/rules.dart

```diff
@@ -0,0 +1,11 @@
+/// Holds game rules for different game variations
+const gameRules = <String, String>{
+  'Traditional':
+      'The first player places an X in any square on the board. Players take turns placing an X or an O in one of the squares as in traditional tic-tac-toe. The first player to get three-in-a-row vertically, horizontally, or diagonally wins.',
```

Comment:
Remove `as in traditional`, because it is implied by name now.

Reply:
fixed